tensorboard/default.py

```diff
@@ -86,4 +87,30 @@ def get_plugins():
 
   :rtype: list[Union[base_plugin.TBLoader, Type[base_plugin.TBPlugin]]]
   """
+
   return _PLUGINS[:]
+
+# TODO(stephanwlee): Combine with get_plugins when concept of first-party
+# plugins is undone.
+def get_dynamic_plugins():
+  """Returns a list specifying TensorBoard's dynamically loaded plugins.
+
+	A dynamic TensorBoard plugin is specified using entry_points [1]. This will be
```

Comment:
Your editor appears to be inserting literal tabs (0x09). Please run
these files through `expand -t 2` and consider adding

```viml
set expandtab
autocmd FileType python setl ts=2 sw=2 sts=2
```

to your vimrc.

Reply:
I use editorconfig plugin and it should technically take care of that though I have seen few cases when my editorconfig is not honored. retabbed.